zombienet/0001-basic-warp-sync/0001-README.md

```diff
@@ -0,0 +1,75 @@
+# Test design
+The `warp-sync` test works on predefined database which is stored in the cloud and
+fetched by the test. `alice` and `bob` nodes are spun up using this database snapshot.
+
+As `warp-sync` requires at least 3 peers, the test spawns the `charlie` node which is `--sync full`.
```

Comment:
Why not spawn all nodes with the predefined database? I don't get why you let one node sync?

Reply:
Did not think of this. Can be done with predefined database also.

Reply:
Done in: [0649849](https://github.com/paritytech/substrate/pull/12675/commits/0649849bd2088cfe774b1444b24a0ac4e388e9d2)